tests/formatter/test_project.py

```diff
@@ -236,7 +344,6 @@ def test_classifier_gt_tox_no_py_ver(fmt: Fmt, tmp_path: Path) -> None:
     [project]
     requires-python=">=3.11"
     classifiers = [
-      "Programming Language :: Python :: 3 :: Only",
```

Comment:
Why remove this?

Reply:
I read it as that what you wanted. 😅

> I don't see value in the 3 only version with 2 EOL. 

I can easily add it back.